src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -740,7 +758,18 @@ await using (var context = InMemoryApplicationDbContext(contextId))
                     }
                 );
                 
-                VerifyAllMocks(methodologyVersionRepository);
+                publicBlobCacheService.Verify(mock => mock.DeleteItem(It.IsAny<AllMethodologiesCacheKey>()),
+                    Times.Once);
+                publicBlobCacheService.Verify(mock => mock.DeleteItem(new PublicationTreeCacheKey(null)),
+                    Times.Once);
+                publicBlobCacheService.Verify(mock => mock.DeleteItem(new PublicationTreeCacheKey(PublicationTreeFilter.AnyData)),
+                    Times.Once);
+                publicBlobCacheService.Verify(mock => mock.DeleteItem(new PublicationTreeCacheKey(PublicationTreeFilter.LatestData)),
+                    Times.Once);
+                publicBlobCacheService.Verify(mock => mock.DeleteItem(It.IsAny<PublicationCacheKey>()),
+                    Times.Once);
+
+                VerifyAllMocks(methodologyVersionRepository, publicBlobCacheService);
```

Comment:
Unless your concerned about this happening more than once then I would change to using strict behaviour, and make these verifications the setups instead.

Anything not setup but invoked will fail due to the strict behaviour.
Anything setup and not invoked will fail the `VerifyAllMocks.

I think this comment applies elsewhere to to your test changes in this PR where I see you are using default behaviour.